golang/testsuite/networks_impl/test_network.go

```diff
@@ -19,54 +19,53 @@ import (
 
 const (
 	datastoreServiceId services.ServiceID = "datastore"
-	apiServiceIdPrefix = "api-"
+	apiServiceIdPrefix                    = "api-"
 
 	waitForStartupDelayMilliseconds = 1000
-	waitForStartupMaxNumPolls = 15
+	waitForStartupMaxNumPolls       = 15
 )
 
 //  A custom Network implementation is intended to make test-writing easier by wrapping low-level
 //    NetworkContext calls with custom higher-level business logic
 type TestNetwork struct {
-	networkCtx            *networks.NetworkContext
-	datastoreServiceImage string
-	apiServiceImage       string
-	datastoreService      *datastore.DatastoreService
-	personModifyingApiService *api.ApiService
-	personRetrievingApiService *api.ApiService
-	nextApiServiceId      int
+	networkCtx                *networks.NetworkContext
+	datastoreServiceImage     string
+	apiServiceImage           string
+	datastoreIPAddress        string
+	datastorePort             int
+	personModifyingApiClient  *api_service_client.APIClient
+	personRetrievingApiClient *api_service_client.APIClient
+	nextApiServiceId          int
 }
 
 func NewTestNetwork(networkCtx *networks.NetworkContext, datastoreServiceImage string, apiServiceImage string) *TestNetwork {
 	return &TestNetwork{
-		networkCtx:            networkCtx,
-		datastoreServiceImage: datastoreServiceImage,
-		apiServiceImage:       apiServiceImage,
-		datastoreService:      nil,
-		personModifyingApiService: nil,
-		personRetrievingApiService: nil,
-		nextApiServiceId:      0,
+		networkCtx:                networkCtx,
+		datastoreServiceImage:     datastoreServiceImage,
+		apiServiceImage:           apiServiceImage,
+		datastoreIPAddress:        "",
+		datastorePort:             0,
+		personModifyingApiClient:  nil,
+		personRetrievingApiClient: nil,
+		nextApiServiceId:          0,
 	}
 }
 
 //  Custom network implementations usually have a "setup" method (possibly parameterized) that is used
 //   in the Test.Setup function of each test
 func (network *TestNetwork) SetupDatastoreAndTwoApis() error {
-	if network.datastoreService != nil {
-		return stacktrace.NewError("Cannot add datastore service to network; datastore already exists!")
-	}
-	if network.personModifyingApiService != nil || network.personRetrievingApiService != nil {
+
```

Comment:
Same comment as above - curious why we're not switching `datastoreService` -> `datastoreClient`

Reply:
Sorry, I don't understand, these lines were removed.
Actually, I renamed this network values to "clients" version
![image](https://user-images.githubusercontent.com/29951623/124929425-df764e80-dfd6-11eb-9367-b242e41dc316.png)

Reply:
I mean, we're losing the error-checking about `datastoreClient` being nil